src/channel.rs

```diff
@@ -267,12 +306,44 @@ impl ChannelBuilder {
     /// Is it permissible to send keepalive pings without any outstanding streams.
     pub fn keepalive_permit_without_calls(mut self, allow: bool) -> ChannelBuilder {
         self.options.insert(
-            OPT_KEEPALIVE_PERMIT_WITHOUT_CALLS,
+            Cow::Borrowed(OPT_KEEPALIVE_PERMIT_WITHOUT_CALLS),
             Options::Integer(allow as usize),
         );
         self
     }
 
+    /// Optimize a channel.
+    ///
+    /// Default is `OptTarget::Blend`.
+    pub fn optimize_for(mut self, target: OptTarget) -> ChannelBuilder {
+        let val = match target {
+            OptTarget::Latency => CString::new("latency"),
+            OptTarget::Blend => CString::new("blend"),
+            OptTarget::Throughput => CString::new("throughput"),
+        };
+        self.options.insert(
+            Cow::Borrowed(OPT_OPTIMIZATION_TARGET),
+            Options::String(val.unwrap()),
+        );
+        self
+    }
+
+    /// Set a raw int configuration.
+    #[doc(hidden)]
+    pub fn raw_cfg_int(mut self, key: CString, val: usize) -> ChannelBuilder {
```

Comment:
Why `#[doc(hidden)]`? I think it would be convenient for users to testing new configuration.

Reply:
This method is only for test purpose, users should use the encapsulated one instead.

Reply:
Please add a comment for why we need these raw functions.